html homepage with h1 and css with h1 style

sentayhu19 opened the pull request:
This req is about H1 on the html and the styles added on the css

sentayhu19 committed:
html homepage with h1 and css with h1 style

keroloussamy commented on README.md:
```diff
@@ -0,0 +1,73 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
```
Please here you can add your project name and a description and the technologies that you used in this project.

keroloussamy commented on README.md:
```diff
@@ -0,0 +1,73 @@
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
And here please you can add your links and delete the other author.

keroloussamy reviewed (changes requested):
Hi @sentayhu19, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] You can make every change in a separate commit.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sentayhu19 committed:
Update README.md

sentayhu19 committed:
Update README.md

sentayhu19 committed:
Update README.md

sentayhu19 commented:
Thanks For Your Response
I have updated the ReadMe.md with my info.



Sincerely

On Tue, Feb 8, 2022, 8:04 PM Kerolous Samy Grace ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Hi @sentayhu19 <https://github.com/sentayhu19>,
>
> Good job so far!
> There are some issues that you still need to work on to go to the next
> project but you are almost there!
> Required Changes ♻️
>
> *Check the comments under the review.*
>
>
> <https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966>
> Optional suggestions
>
> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
> the approval of this PR. However, I strongly recommend you to take them
> into account as they can make your code better.*
>
> You can also consider:
>
>    - [OPTIONAL] You can make every change in a separate commit.
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
>
> *Please, do not open a new Pull Request for re-reviews. You should use the
> same Pull Request submitted for the first review, either valid or invalid
> unless it is requested otherwise.*
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In README.md
> <https://github.com/sentayhu19/Hello-Microverse/pull/1#discussion_r801858744>
> :
>
> > +# Project Name
>
> +
>
> +> Description the project.
>
> +
>
> +
>
> +## Built With
>
> +
>
> +- Major languages
>
> +- Frameworks
>
> +- Technologies used
>
>
> Please here you can add your project name and a description and the
> technologies that you used in this project.
> ------------------------------
>
> In README.md
> <https://github.com/sentayhu19/Hello-Microverse/pull/1#discussion_r801860324>
> :
>
> > +## Authors
>
> +
>
> +👤 **Author1**
>
> +
>
> +- GitHub: ***@***.***(https://github.com/githubhandle)
>
> +- Twitter: ***@***.***(https://twitter.com/twitterhandle)
>
> +- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
>
> +
>
> +👤 **Author2**
>
> +
>
> +- GitHub: ***@***.***(https://github.com/githubhandle)
>
> +- Twitter: ***@***.***(https://twitter.com/twitterhandle)
>
> +- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
>
>
> And here please you can add your links and delete the other author.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/sentayhu19/Hello-Microverse/pull/1#pullrequestreview-876356180>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASRE4335VS5X7EGTOFII4ATU2FEKNANCNFSM5N3AXDYQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

sentayhu19 commented:
@keroloussamy
Thanks For Your Response
I have updated the ReadMe.md with my info.

Sincerely

On Tue, Feb 8, 2022 at 9:08 PM Sentayhu Berhanu ***@***.***>
wrote:

> Thanks For Your Response
> I have updated the ReadMe.md with my info.
>
>
>
> Sincerely
>
> On Tue, Feb 8, 2022, 8:04 PM Kerolous Samy Grace ***@***.***>
> wrote:
>
>> ***@***.**** requested changes on this pull request.
>>
>> Hi @sentayhu19 <https://github.com/sentayhu19>,
>>
>> Good job so far!
>> There are some issues that you still need to work on to go to the next
>> project but you are almost there!
>> Required Changes ♻️
>>
>> *Check the comments under the review.*
>>
>>
>> <https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966>
>> Optional suggestions
>>
>> *Every comment with the [OPTIONAL] prefix is not crucial enough to stop
>> the approval of this PR. However, I strongly recommend you to take them
>> into account as they can make your code better.*
>>
>> You can also consider:
>>
>>    - [OPTIONAL] You can make every change in a separate commit.
>>
>> Cheers and Happy coding!👏👏👏
>>
>> Feel free to leave any questions or comments in the PR thread if
>> something is not 100% clear.
>> Please, remember to tag me in your question so I can receive the
>> notification.
>>
>> *Please, do not open a new Pull Request for re-reviews. You should use
>> the same Pull Request submitted for the first review, either valid or
>> invalid unless it is requested otherwise.*
>> ------------------------------
>>
>> *As described in the Code reviews limits policy
>> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
>> limited number of reviews per project (check the exact number in your
>> Dashboard). If you think that the code review was not fair, you can request
>> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
>> ------------------------------
>>
>> In README.md
>> <https://github.com/sentayhu19/Hello-Microverse/pull/1#discussion_r801858744>
>> :
>>
>> > +# Project Name
>>
>> +
>>
>> +> Description the project.
>>
>> +
>>
>> +
>>
>> +## Built With
>>
>> +
>>
>> +- Major languages
>>
>> +- Frameworks
>>
>> +- Technologies used
>>
>>
>> Please here you can add your project name and a description and the
>> technologies that you used in this project.
>> ------------------------------
>>
>> In README.md
>> <https://github.com/sentayhu19/Hello-Microverse/pull/1#discussion_r801860324>
>> :
>>
>> > +## Authors
>>
>> +
>>
>> +👤 **Author1**
>>
>> +
>>
>> +- GitHub: ***@***.***(https://github.com/githubhandle)
>>
>> +- Twitter: ***@***.***(https://twitter.com/twitterhandle)
>>
>> +- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
>>
>> +
>>
>> +👤 **Author2**
>>
>> +
>>
>> +- GitHub: ***@***.***(https://github.com/githubhandle)
>>
>> +- Twitter: ***@***.***(https://twitter.com/twitterhandle)
>>
>> +- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
>>
>>
>> And here please you can add your links and delete the other author.
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/sentayhu19/Hello-Microverse/pull/1#pullrequestreview-876356180>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ASRE4335VS5X7EGTOFII4ATU2FEKNANCNFSM5N3AXDYQ>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

keroloussamy commented:
Hello @sentayhu19, when you change the requirements submit the same pull request again in your dashboard so any reviewer will review it for you.

sentayhu19 committed:
Update README.md

BenMukebo commented on README.md:
```diff
@@ -0,0 +1,71 @@
+##### from parent source directory
+Copyright (c) 2022 Sentayhu
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
```
- [OPTIONAL ] No need of enumerating this license agreement, I would suggest you remove them.

BenMukebo reviewed (approved):
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @Sentayhu Robi** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
